@here/harp-mapview/lib/MapView.ts

```diff
@@ -795,6 +808,9 @@ export class MapView extends THREE.EventDispatcher {
 
     private m_enableMixedLod: boolean | undefined;
 
+    private m_tmpVector1 = new THREE.Vector3();
```

Comment:
there is already a `cache` for vectors at the top of the file

Reply:
Didn't know this, thanks.